### PR TITLE
feat: flashloans docs refactoring

### PR DESCRIPTION
**Pull request by sotnikov-s**
This PR replaces code and text duplications related to `neutron-flashloans` with links to source code

### CODE REVIEW

File: docs/other_tools/flashloans/overview.md [hunk 1]
```diff
@@ -1,64 +1,16 @@
...
-  message is submitted as a submessage with a `reply_on_success` strategy, meaning if it fails, the transaction is
-  reverted.
-* Upon successful execution of the `/cosmos.bank.v1beta1.MsgSend` message, the `neutron-flashloans` contract sends
-  a `ProcessLoan` submessage with a `reply_on_success` strategy to the borrower contract.
-* After receiving a successful reply to the `ProcessLoan` message, the `neutron-flashloans` contract verifies that the
-  borrower has returned the funds and paid the fee, then it deletes the loan information.
+### Neutron flashloans
 
-## Security advice
+The `neutron-flashloans` contract facilitates providing flash loans to smart contracts operating on the Neutron network. The `neutron-flashloans` contract does not hold any funds. Instead, it uses `authz` permission from the `source` address to execute `/cosmos.bank.v1beta1.MsgSend` on its behalf. For this particular instance, the `source` address is the [Treasury (Neutron DAO core)](https://neutron.celat.one/neutron-1/contracts/neutron1suhgf5svhu4usrurvxzlgn54ksxmn8gljarjtxqnapv8kjnp4nrstdxvff) contract address.
```
Comment (pr0n00gler) [inline docs/other_tools/flashloans/overview.md:13]:
```suggestion
The `neutron-flashloans` contract facilitates provision of flash loans to smart contracts operating on the Neutron network. The `neutron-flashloans` contract does not hold any funds. Instead, it uses `authz` permission from the `source` address to execute `/cosmos.bank.v1beta1.MsgSend` on its behalf. For this particular instance, the `source` address is the [Treasury (Neutron DAO core)](https://neutron.celat.one/neutron-1/contracts/neutron1suhgf5svhu4usrurvxzlgn54ksxmn8gljarjtxqnapv8kjnp4nrstdxvff) contract address.
```

Sounds better in my opinion, not an expert in English though

File: docs/other_tools/flashloans/overview.md [hunk 2]
```diff
@@ -1,64 +1,16 @@
...
 
-When writing a borrower contract, ensure that the `ProcessLoan` handler has proper permissions. It should only be
-callable when your contract has previously requested a loan and only by the `neutron-flashloans` contract.
+- **contract address**: TODO add celatone link to the contract instance when deployed
+- **usage**: https://github.com/neutron-org/neutron-dao/blob/main/contracts/dao/neutron-flashloans
```
[the remark this answers is not in the document]
Comment (pr0n00gler) [reply docs/other_tools/flashloans/overview.md:16]: Maybe we should give a link to an example contract that uses the flashloan contract instead? (cause you say **usage**).

And have a separate link to the actual implementation of Flashloan contract that leads to crate.io

[the remark this answers is not in the document]
Comment (sotnikov-s) [reply docs/other_tools/flashloans/overview.md:16]: I got your point. I'll see how to improve it and let you know

[the remark this answers is not in the document]
Comment (sotnikov-s) [reply docs/other_tools/flashloans/overview.md]: added the link to the neutron-dao PR cuz find it the proper place for it: https://github.com/neutron-org/neutron-dao/pull/108

**Pull request closed without merging**